src/workerd/api/tests/js-rpc-test.js

```diff
@@ -1006,7 +1006,24 @@ export let streams = {
       await promise;
     }
 
-    // TODO(someday): Test JS-backed WritableStream, when it actually works.
+    {
+      const dec = new TextDecoder();
+      let result = '';
+      const { promise, resolve } = Promise.withResolvers();
+      const writable = new WritableStream({
+        write(chunk) {
+          result += dec.decode(chunk, {stream: true});
+        },
+        close() {
+          result += dec.decode();
+          resolve();
+        },
```

Comment:
For review... once we figure out the right approach for aborting the writable stream such that the abort algorithm is invoked reliably, we need to add a test that verifies it.